cumulus/test/runtime/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+use super::{
+	AccountId, AuraConfig, AuraId, BalancesConfig, ParachainInfoConfig, RuntimeGenesisConfig,
+	SudoConfig,
+};
+use alloc::{vec, vec::Vec};
+
+use cumulus_primitives_core::ParaId;
+use frame_support::build_struct_json_patch;
+use sp_genesis_builder::PresetId;
+use sp_keyring::Sr25519Keyring;
+
+fn cumulus_test_runtime(
+	invulnerables: Vec<AuraId>,
+	endowed_accounts: Vec<AccountId>,
+	id: ParaId,
+) -> serde_json::Value {
+	let config = build_struct_json_patch!(RuntimeGenesisConfig {
```

Comment:
Macro already provides `serde_json::Value`:
```suggestion
	build_struct_json_patch!(RuntimeGenesisConfig {
```